homeassistant/components/media_player/kodi.py

```diff
@@ -155,7 +150,17 @@ def media_duration(self):
     def media_image_url(self):
         """ Image url of current playing media. """
         if self._item is not None:
-            return _get_image_url(self._item['thumbnail'])
+            return self._get_image_url()
+
+    def _get_image_url(self):
+        """ Helper function that parses the thumbnail URLs used by Kodi. """
+        url_components = urllib.parse.urlparse(self._item['thumbnail'])
+        
+        if url_components.scheme == 'image':
+            return\
+                self._url.split('/jsonrpc')[0] +\
+                "/image/" +\
+                urllib.parse.quote_plus(self._item['thumbnail'])
```

Comment:
Yes. That was a bit tricky to figure out.

The _item['thumbnail'] contains a string that looks something like 
image://<ur encoded path>?size:thumb  
The unemcoded parts needs to be encoded.